build.gradle

```diff
@@ -14,7 +14,7 @@ repositories {
     jcenter()
 }
 
-def compileAvroVersion = "1.10.0"
+def compileAvroVersion = "1.10.1"
```

Comment:
Please add "1.10.1" to `def avroVersions` on line 210, to support runtime compatibility testing of both versions.